backend/src/routes/account.js

```diff
@@ -70,6 +70,8 @@ router.post('/register',
     try {
       let createdUser = new User(req.body.user)
 
+      await createdUser.sendVerificationEmail()
```

Comment:
we should put this line after User.register, and we should remove await

Reply:
👌